src/streamlink_cli/main.py

```diff
@@ -886,12 +885,12 @@ def setup_plugin_options(session, plugin):
 def check_root():
     if hasattr(os, "getuid"):
         if os.geteuid() == 0:
-            console.logger.info("streamlink is running as root! Be careful!")
+            log.info("streamlink is running as root! Be careful!")
 
 
 def log_current_versions():
     """Show current installed versions"""
-    if args.loglevel == "debug":
+    if logger.root.isEnabledFor(logging.DEBUG):
```

Comment:
@beardypig I found a small issue

when `loglevel=debug` is setup in the config file,
it won't print `log_current_versions()` because `logger.root` won't change

```text
$ streamlink
>>> setup_args(parser, ignore_unknown=True)
---- args.loglevel
info
---- logger.root
<Logger streamlink (warning)>
>>> setup_config_args(parser)
---- args.loglevel
debug
---- logger.root
<Logger streamlink (info)>
>>> log_current_versions()
---- args.loglevel
debug
---- logger.root
<Logger streamlink (info)>
```

Reply:
Good catch @back-to, I'm taking a look now.

Reply:
Opened #1715 which resolves it for me, can you take a look?